### ISSUE TITLE
Add capabilties to use crtmqm flags for such as (-fs, -lp, -ls , -lc , -lf,  -h,  -u, -p)

### ISSUE THREAD

**Issue by MQGIT** [ibm-messaging/mq-ansible#106]
**Is your feature request related to a problem? Please describe.**
As part of the create queue manager task, there must be a way to configure the queue manager at creation time with the file system size, the number of primary secondary logss, logs file sizes, and so on. 

**Describe the solution you'd like**
I'd like to be able to configure the queue manager at creation time with all the parameters I require for my environment. 

**Describe alternatives you've considered**
Not to use the create queue manager role and resort to using shell commands to create the queue managers that way I need them.

**Additional context**
This also falls inline with adding RDQM capabilities. As of today, my team and I have developed several roles to build an RDQM environment. I'd have loved to see this as part of the mq-ansible repo.

**Comment by kengay21** [comment]
I agree that there should be the capability to create queue managers with these flags - in addition to those needed to create RDQM type queue managers.  I am just getting into learning Ansible, so I am far from being qualified to provide any kind of meaningful input.  But, could variables be set for each of these flags that have null values if they are not needed and if needed, set with both the flag and the value, e.g. "lfsize: -lf 65535", if needed or "lfsize: ", if not needed?  And then the crtmqm command would look something like "crtmqm {{ lfsize }} QM1.  That way the user has full control in determining when a non-default flag is used.

**Comment by spencer-cdw** [comment]
This has been added in a draft PR here: https://github.com/ibm-messaging/mq-ansible/pull/123
